libs/pages/environments/src/lib/feature/create-clone-environment-modal-feature/create-clone-environment-modal-feature.tsx

```diff
@@ -36,6 +39,10 @@ export function CreateCloneEnvironmentModalFeature(props: CreateCloneEnvironment
     },
   })
 
+  methods.watch((data) => {
+    setMustConfirmClickOutside(methods.formState.isDirty)
```

Comment:
I think the naming should be `setAlertModalClickOutside` because you use this name for your component it's more ISO, wdyt ?